snuba/admin/static/runtime_config/types.tsx

```diff
@@ -8,4 +8,12 @@ type Config = { key: ConfigKey; value: ConfigValue; type: ConfigType };
 
 type RowData = [ReactNode, ReactNode, ReactNode];
 
-export { Config, ConfigKey, ConfigValue, ConfigType, RowData };
+type ConfigChange = {
+  key: ConfigKey;
+  user: string | null;
+  timestamp: number;
+  before: ConfigValue | null;
```

Comment:
I think the canonical optional syntax in TS uses the `?`: https://stackoverflow.com/a/47942746

```suggestion
  before?: ConfigValue;
```

this lets you use the nice null-safe accesses like `configChangeInstance.before?.configValueProperty` which will be null if before is null

Reply:
This is the typing of the data that is returned from the API. The keys `before` and `after` are always present but they can be null. The question mark implies it can be missing, which I don't think is the case, so I prefer the stricter way.